js/src/edit.js

```diff
@@ -55,7 +55,7 @@ function configureStore() {
 function wrapInTopLevelComponents( Component, store ) {
 	return (
 		<IntlProvider
-			locale={ localizedData.intl.locale }
+			locale={ localizedData.intl.locale.substr( 0, 2 ).toLowerCase() }
```

Comment:
Not sure about the need of this change (not arguing, I just don't know the reason)

Reply:
Merging for now. @Xyfi Please address this when you're back.